mstyle/mstyle.cpp

```diff
@@ -1290,8 +1290,8 @@ bool MgStyle::drawFrameGroupBoxPrimitive(const QStyleOption* option, QPainter* p
             return true;
 
       // no frame for flat groupboxes
-      QStyleOptionFrameV2 fOpt2(*fOpt);
-      if (fOpt2.features & QStyleOptionFrameV2::Flat)
+      QStyleOptionFrame fOpt2(*fOpt);
+      if (fOpt2.features & QStyleOptionFrame::Flat)
```

Comment:
I'm still not sure about QStyleOptionFrame. The comment in the link I posted I think only refers to `initStyleOption`, not `QStyleOptionFrame`
Unfortunately I cannot check older qt versions here, but what I found is that qt542 contains all three QStyleOptionFrame QStyleOptionFrameV2 and QStyleOptionFrameV3.
That travis was able to build the commit does not tell us anything? Will it only be a runtime problem?
Would you in any case just prefer to keep the old compat declarations?